tasks/sync_packages.js

```diff
@@ -7,6 +7,8 @@ var glob = require('glob');
 var common = require('./util/common');
 var constants = require('./util/constants');
 var pkg = require('../package.json');
+var rc = pkg.version.split('-')[1];
```

Comment:
won't this error out if it's a non-RC version string like `2.0.1` ?

Reply:
No I don't think so. It was tested. In that case the tag and variable `rc` would be undefined.

Reply:
Oh Javascript :) `"string".split("-")[1] -> undefined` ... ok then! I've been doing Python too long!